### PR TITLE
Update 1.20.1, inactive and more

### PR DESCRIPTION
**Pull request by TonimatasDEV**
- Add the missing points in the descriptions.
 - Upgrade to 1.20.1.
 - Move all software that have not had any commit in the last year to Inactive.
- Correct some typos.
- Fix some RECOMMEND.md links.

Close #50

### CODE REVIEW

File: RECOMMEND.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+  - If you want to use plugins on your **1.8-1.18** server, use [Paper Legacy](/java/PLUGINS.md#%EF%B8%8F-paper).
+  - If you want to use plugins on your **1.7.10** server, use [PaperSpigot](/java/PLUGINS.md#%EF%B8%8F-paper).
   - If you want to use plugins on **Beta 1.7.3** use [Canyon](/java/PLUGINS.md#-canyon).
   
 ### 🔨 Minecraft with mods
-  - If you want to use mods on your server you can use [Forge](/java/MODS.md#-forge) or [Fabric](/java/MODS.md#-fabric), depending which mods you want to use.  
+  - If you want to use mods on your server, you can use [Forge](/java/MODS.md#-forge) or [Fabric](/java/MODS.md#-fabric), depending on which mods you want to use.  
   
 ### 🔶 Minecraft with mods and plugins
+  - If you want to use Forge mods and plugins on **1.16.5** you can use [Mohist](/java/MODS+PLUGINS.md#-mohist).
```
Comment (UeberallGebannt) [inline RECOMMEND.md:20]: 1.16.5 already exists as an entry

File: bedrock/PLUGINS.md [hunk 2]
```diff
@@ -51,20 +37,34 @@ Since most of the softwares do not support all vanilla features, it's recommende
...
 
 ### [✳ MiNET](https://github.com/NiclasOlofsson/MiNET)
 - **Author:** NiclasOlofsson
 - **Fork:** -
-- **Description:**  A (not so) basic Minecraft Pocket Edition server written in C# 
+- **Description:** A (not so) basic Minecraft Pocket Edition server written in C#.
 - **Plugins:** .NET
 - **Addons:** ❌
+
+### [💨 LiteLoader BDS](https://github.com/LiteLDev/LiteLoaderBDS)
```
Comment (UeberallGebannt) [inline bedrock/PLUGINS.md:58]: Why did you move this into inactive, it is active.
grafik

File: java/PLUGINS.md [hunk 3]
```diff
@@ -254,47 +253,47 @@ This list contains Minecraft Java plugins server softwares.
...
 - **Alternative:** Purpur/Pufferfish
 - **Version:** 1.19.2
 - **Author:** PrismarineTeam
 - **Fork:** CraftBukkit --> Spigot --> Paper --> Pufferfish/Purpur/Jettpack/EmpireCraft/Patina/Mirai/Suki/Akarin/... --> Prismarine
-- **Description:** Fork of Paper that combines the patches from different forks (Note: not recommended and abandoned)
+- **Description:** Fork of Paper that combines the patches from different forks (Note: not recommended and abandoned).
 - **Plugins:** Bukkit, Spigot, Paper
 
-### [⬛️ Yatopia](https://github.com/YatopiaMC/Yatopia)
+### [⬛️ Topiary](https://github.com/YatopiaMC/Yatopia)
```
Comment (UeberallGebannt) [inline java/PLUGINS.md:283]: Why did you rename this?

**Review by UeberallGebannt — CHANGES_REQUESTED** [review]

File: bedrock/PLUGINS.md (hunk 2 above)
Comment (TonimatasDEV) [reply bedrock/PLUGINS.md]: My bad, is Cloudburst not LiteLoaderBDS

File: RECOMMEND.md (hunk 1 above)
Comment (TonimatasDEV) [reply RECOMMEND.md]: Sorry, 1.20.1, already fixed.

File: java/PLUGINS.md (hunk 3 above)
Comment (TonimatasDEV) [reply java/PLUGINS.md]: mmm? I don't know what happened there but it's fixed now.

**Comment by TonimatasDEV** [comment]
@UeberallGebannt All you request are done

**Review by UeberallGebannt — APPROVED** [review]
LGTM!

**Pull request merged**